src/components/completing-party/CompletingParty.vue

```diff
@@ -147,7 +147,7 @@
 import Vue from 'vue'
 import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
 import { AddressIF, CompletingPartyIF, FormIF } from '@bcrs-shared-components/interfaces/'
-import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+import BaseAddress from '../base-address/BaseAddress.vue'
```

Comment:
Change this to use the shared BaseAddress component. If not then when THIS component is used in the host app then it will look for BaseAddress using the relative path above, and fail.

Also add base-address as a dependency (in package file).